src/main/scala/com/databricks/spark/csv/DefaultSource.scala

```diff
@@ -125,6 +125,16 @@ class DefaultSource
       throw new Exception("Treat empty values as null flag can be true or false")
     }
 
+    val treatParseExceptionAsNull = parameters.getOrElse(
+      "treatParseExceptionAsNull", "false")
```

Comment:
I suggest changing name of this option to something like `insertNullOnErrors`. Primarily because this data source is used by R and Python users and exceptions are not familiar concepts for them.